cypress/fixtures/test-data.template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "email": "${CYPRESS_TEST_USER_EMAIL}",
+  "verify_code": "7fd76efb-9ac4-4627-b0b3-3de6a01d07fb",
```

Comment:
is this the designstore's token? might be good to have it as an environment variable as well 🤔

Reply:
Done